src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,34 @@
+import {useParams} from "react-router-dom"
+import { getArticle, getAuthorName } from "../../utils/utils"
+import { useEffect, useState } from "react"
+import './articlePage.css'
+
+export default function ArticlePage () {
+    const {article_id} = useParams()
+    const [articleInfo, setArticleInfo] = useState([])
+    const [authorName, setAuthorName] = useState("")
+     
+    useEffect(() => {
+        getArticle(article_id).then((response) => {
+            setArticleInfo(response)
+            return getAuthorName(response.author)
```

Comment:
i'm pretty sure the article has the author on it so you don't need to grab the author name like you are. also you can potentially add loading functionality for user feedback